include/SDL3/SDL_metal.h

```diff
@@ -73,6 +74,8 @@ extern DECLSPEC SDL_MetalView SDLCALL SDL_Metal_CreateView(SDL_Window * window);
  * This should be called before SDL_DestroyWindow, if SDL_Metal_CreateView was
  * called after SDL_CreateWindow.
  *
+ * \param view the view
```

Comment:
Maybe change to `The SDL_MetalView object` ??

Reply:
done in https://github.com/libsdl-org/SDL/pull/7300